Backend/StudentService.cs

```diff
@@ -0,0 +1,30 @@
+using FRITeam.Swapify.Backend.Interfaces;
+using FRITeam.Swapify.Entities;
+using MongoDB.Driver;
+using System;
+using System.Threading.Tasks;
+
+namespace FRITeam.Swapify.Backend
+{
+    public class StudentService : IStudentService
+    {
+        private readonly IMongoDatabase _database;
+
+        public StudentService(IMongoDatabase database)
+        {
+            _database = database;
+        }
+
+        public async Task AddStudentAsync(Student entityToAdd)
+        {
+            entityToAdd.Id = Guid.NewGuid();
+            await _database.GetCollection<Student>(nameof(Student)).InsertOneAsync(entityToAdd);
+        }
+
+        public async Task<Student> FindStudentById(string guid)
```

Comment:
1. We should not repeat Student in method name... Just AddAsync and FindAsync.
2. parameter name is not guid, but Id and type is not string but Guid

Reply:
ok